src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel ="stylesheet" href="https://fonts.google.com/specimen/Roboto">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png"
+        alt="Moyo"
+        class="logo_img">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
```

Comment:
```suggestion
          <li class="nav_item">
            <a href="#apple" class="nav_link is-active">Apple</a>
          </li>
          
          <li class="nav_item">
```

Add an empty line between multiple sibling blocks of html